src/domain/visualization/entities/Visualization.ts

```diff
@@ -0,0 +1,17 @@
+import { Id } from "../../common/entities/Schemas";
+
+type VisualizationRowItem = {
+    id: string;
+    name: string;
+};
+
+type VisualizationRow = {
+    id: string;
+    dimension: string;
+    items: VisualizationRowItem[];
+};
+
+export type Visualization = {
```

Comment:
minor: top-down types is usually more readable, the "important" type is the first you see when you open the file.